### PR TITLE
Test clean up

### PR DESCRIPTION
**Pull request by eevans**
@hardikj here are some additional suggestions; It seemed easier to iterate on these locally than to try to add them to https://github.com/wikimedia/restbase-mod-table-cassandra/pull/110 individually.

Let me know if you have any questions.

### CODE REVIEW

File: test/utils/test_router.js [hunk 1]
```diff
@@ -95,7 +94,7 @@ router.setup = function(_options) {
         });
     })
     .catch(function(e){console.log(e);});
-}
+};
```
Comment (hxjuneja) [inline test/utils/test_router.js:97]: Don't why jshint don't lint this on my system

**Comment by hxjuneja** [comment]
@eevans thanks for extra efforts
seems like I ruined the "test description" strings there or maybe didn't got what you meant.

**Comment by eevans** [comment]
> @eevans thanks for extra efforts
> seems like I ruined the "test description" strings there or maybe didn't got what you meant.

I had a round of suggestions after we had already iterated (like I said, bikeshedding), and I didn't want to burn up any more of either of our time than was necessary (it would have taken me long to enter line comments in Github).

TL;DR I appreciate your efforts on this!
